### PR TITLE
Update r

### PR DESCRIPTION
**Pull request by douglasday**

### CODE REVIEW

File: r [hunk 1]
```diff
@@ -1 +1 @@
-r
```
Comment (douglasday) [inline r:1]: bro

**Review by douglasday — COMMENTED** [review]
yo

File: r (hunk 1 above)
Comment (jon-h1test-admin) [reply r:1]: test @douglasday

Comment (jon-h1test-admin) [reply r:1]: danger

File: r [hunk 2]
```diff
@@ -1 +1 @@
...
+2
```
Comment (jon-h1test-admin) [inline r:1]: @douglasday fix this

**Review by jon-h1test-admin — COMMENTED** [review]
@douglasday plese fix

**Pull request merged**